src/controllers/room.controller.js

```diff
@@ -0,0 +1,137 @@
+const bcrypt = require('bcryptjs')
+const parametros = require('../lib/configParameters')
+const mongoose =  require("mongoose");
+const { v4: uuidv4 } = require("uuid");
+
+const mongodbConnection = require('../databases/mongodb/repository/mongodbManager');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Backend/pullRequest?prid=7172443)
```suggestion
const mongodbConnection = require("../databases/mongodb/repository/mongodbManager");
```